TerrorConsole/Assets/_Main/Scripts/Dialogue/DialogueData.cs

```diff
@@ -0,0 +1,11 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    [System.Serializable]
+    public class DialogueData 
+    {
+        [TextArea(3, 10)]
+        public string[] sentences;
+    }
+}
```

Comment:
This class must change cause we are going to need all dialogues be code driven, we can change it together.

Reply:
Nevermind, changed my mind.